warehouse/query-core/src/test/java/datawave/query/jexl/JexlASTHelperTest.java

```diff
@@ -122,6 +125,18 @@ public void test4() throws Exception {
         }
     }
     
+    @Test
+    public void testGetLiteralValueSafely() throws Exception {
+        ASTJexlScript query = JexlASTHelper.parseJexlQuery("FOO == BAR");
```

Comment:
This is a nitpick, I would like to see "FOO == FOO2", given how frequently we use "FOO == 'bar'"